### DISCUSSION TITLE
What's the relationship between vsomip and some/ip protocol version?

### DISCUSSION THREAD

**Discussion by emptyx0** [COVESA/vsomeip#786] [Q&A] [answered]
Vsomeip implements the some/ip protocol which has many versions, ex. RS23-11, RS22-11 ,..., so how can i know the mapping between vsomeip's version and protocol version

**Comment by duartenfonseca** [comment, marked as answer]
There is no direct link between the vsomeip version and the someip protocol used. we try to update the vsomeip implementation to the most current someip protocol version.
